Bridge/Bridge.sol

```diff
@@ -245,7 +245,8 @@ contract Bridge is EthTokenReciever {
         require(
             checkSignatures(
                 keccak256(
-                    abi.encodePacked(
```

Comment:
Transition from encode to encodePacked is not that necessary but might be taken as it may only affect the order of the params for the statement we sign but not the signature itself
Just make sure that it works fine when passing params to recoverAddress, now it looks okay, but it will not be superfluous to double check it

Reply:
It's needed to prevent signature collision